lumen/ai/utils.py

```diff
@@ -135,7 +135,8 @@ def format_float(num):
         if isinstance(df[col].iloc[0], pd.Timestamp):
             df[col] = pd.to_datetime(df[col])
 
-    df_describe_dict = df.describe(percentiles=[]).drop(["min", "max"]).to_dict()
+    df_describe_dict = df.describe(percentiles=[]).drop(
+        ["min", "max"], errors="ignore").to_dict()
```

Comment:
```suggestion      
    describe_df = df.describe(percentiles=[])
    columns_to_drop = ["min", "max"] # present if any numeric
    columns_to_drop = [col for col in columns_to_drop if col in describe_df.columns]
    df_describe_dict = describe_df.drop(columns=columns_to_drop).to_dict()
```

It might be a bit safer not to ignore all errors and to check if these specific cols exist